internal/pkg/flink/internal/store/store_test.go

```diff
@@ -291,107 +291,161 @@ func (s *StoreTestSuite) TestIsExitStatement() {
 	assert.Equal(s.T(), "value", value)
+	assert.Nil(s.T(), err)
 
-	key, value, _ = parseSetStatement("set 'key    '= '		va  lue'    ")
-	assert.Equal(s.T(), "key", key)
-	assert.Equal(s.T(), "value", value)
+	key, value, err = parseSetStatement("set 'key    '= '		va  lue'    ")
+	assert.Equal(s.T(), "key    ", key)
+	assert.Equal(s.T(), "\t\tva  lue", value)
+	assert.Nil(s.T(), err)
 
-	key, value, _ = parseSetStatement("set 'key' ='value'    ")
+	key, value, err = parseSetStatement("set 'key' ='value'    ")
 	assert.Equal(s.T(), "key", key)
 	assert.Equal(s.T(), "value", value)
+	assert.Nil(s.T(), err)
 
-	key, value, _ = parseSetStatement("set 'key'		 ='value'    ")
+	key, value, err = parseSetStatement("set 'key'		 ='value'    ")
 	assert.Equal(s.T(), "key", key)
 	assert.Equal(s.T(), "value", value)
+	assert.Nil(s.T(), err)
 
-	key, value, _ = parseSetStatement("set")
+	key, value, err = parseSetStatement("set")
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
+	assert.Nil(s.T(), err)
 
-	key, value, _ = parseSetStatement("SET")
+	key, value, err = parseSetStatement("SET")
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
+	assert.Nil(s.T(), err)
 
-	key, value, _ = parseSetStatement("sET 	")
+	key, value, err = parseSetStatement("sET 	")
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
+	assert.Nil(s.T(), err)
 
-	key, value, _ = parseSetStatement("sET 'key'	")
+	key, value, err = parseSetStatement("sET 'key'	")
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
+	assert.Error(s.T(), err)
 
-	key, value, _ = parseSetStatement("sET = 'value'	")
+	key, value, err = parseSetStatement("sET = 'value'	")
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
+	assert.Error(s.T(), err)
 
-	key, value, _ = parseSetStatement("sET 'key'= \n'value'	")
+	key, value, err = parseSetStatement("sET 'key'= \n'value'	")
 	assert.Equal(s.T(), "key", key)
 	assert.Equal(s.T(), "value", value)
+	assert.Nil(s.T(), err)
+
+	key, value, err = parseSetStatement("set key= \nvalue	")
+	assert.Equal(s.T(), "", key)
+	assert.Equal(s.T(), "", value)
+	assert.Error(s.T(), err)
 
-	key, value, _ = parseSetStatement("set key= \nvalue	")
+	key, value, err = parseSetStatement("set 'key'= \nvalue	")
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
+	assert.Error(s.T(), err)
 
-	key, value, _ = parseSetStatement("set 'key'= \nvalue	")
+	key, value, err = parseSetStatement("set key= \n'value'	")
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
+	assert.Error(s.T(), err)
 
-	key, value, _ = parseSetStatement("set key= \n'value'	")
+	key, value, err = parseSetStatement("set 'key= \nvalue'	")
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
+	assert.Error(s.T(), err)
 
-	key, value, _ = parseSetStatement("set 'key= \nvalue'	")
+	key, value, err = parseSetStatement(`set ''key''=''value''`)
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
```

Comment:
I think what I was alluding to earlier is, good Go code shouldn't care what the other values are equal to if a function returns an error. It seems weird to be checking these values if we'll never use them in the case of an error.

Reply:
Oh I see, yeah I guess we could just remove those checks, but they probably don't hurt either.